src/ensembl/src/shared/components/communication-framework/contact-us/ContactUs.tsx

```diff
@@ -29,13 +36,63 @@ const ContactUs = () => {
     @Imran: update the below accordingly
   */
   return (
-    <div>
-      <p>
-        Please contact us if you have a problem with the website or need help
-      </p>
-      <SecondaryButton onClick={() => setShouldShowForm(!shouldShowForm)}>
-        Contact us
-      </SecondaryButton>
+    <div className={styles.wrapper}>
+      <div>
+        <p>
+          Please contact us if you have a problem with the website or need help
+        </p>
+        <SecondaryButton onClick={() => setShouldShowForm(!shouldShowForm)}>
+          Contact us
+        </SecondaryButton>
+      </div>
+      <div>
+        <section className={styles.socialMediaInfo}>
+          <p>
+            Visit the blog for details of releases, workshops and other tidbits
+            of information about the Ensembl project.
+          </p>
+          <p>
+            Visit Facebook or Twitter for the latest breaking Ensembl news and
+            service status updates.
+          </p>
+          <p>
+            Ensembl Blog <BlogIcon className={styles.icon} />{' '}
+            <FacebookIcon className={styles.icon} />{' '}
+            <TwitterIcon className={styles.icon} />
+          </p>
+        </section>
+        <section>
+          <h4>Online Help</h4>
+          <p>
+            Each app has a Help icon (top right) that will show
+            context-sensitive help for that view.
+          </p>
+          <p>All Help articles and videos can be found in the Help app.</p>
+        </section>
+        <section>
+          <h4>Mailing lists</h4>
+          <p> We have two public mailing lists: </p>
+          <dl>
```

Comment:
Why do you use the description list tags? Semantically, description lists are intended for things like glossaries, etc (see [html5 spec](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-dl-element.html))

Reply:
I went by the MDN description of them:

```The <dl> HTML element represents a description list. The element encloses a list of groups of terms (specified using the <dt> element) and descriptions (provided by <dd> elements). Common uses for this element are to implement a glossary or to display metadata (a list of key-value pairs).```

Since we are specifying key-value pairs here, thought `dl` is better than other list types.

Reply:
Hm. All right then.